text/0034-telemetry-policies.md

```diff
@@ -46,6 +46,7 @@ The collected data is sent to [Segment](https://segment.com/). Segment is a plat
 | Documents Searched GET | Aggregated event on all received requests via the `GET` - `/indexes/:indexUid/search` route during one hour or until a batch size reaches `500Kb`. |
 | Documents Added | Aggregated event on all received requests via the `POST` - `/indexes/:indexUid/documents` route during one hour or until a batch size reaches `500Kb`. |
 | Documents Updated | Aggregated event on all received requests via the `PUT` - `/indexes/:indexUid/documents` route during one hour or until a batch size reaches `500Kb`. |
+| Documents Deleted | Aggregated event on all received requests via the `DELETE` - `/indexes/:indexUid/documents`, `DELETE` - `/indexes/:indexUid/documents/:documentId`, `POST` - `indexes/:indexUid/documents/delete-batch` routes during one hour or until a batch size reaches `500Kb`.  |
```

Comment:
Super meta comment.
Do you think we should start to send analytics on our analytics?
Like, send an event when a batch_size reaches its limit?
That would allow us to decide if we need to increase the batch size (and risk bothering the small machine) or send the analytics more often (and start getting messages from machines running in a CI for a single test).

It's not super easy to code since it's located in the segment library, but I think I could do it in a few days :thinking:

Reply:
I think it's ok for now!